backend/app/api/health.py

```
@@ -214,6 +216,66 @@ async def health_alias(response: Response) -> dict:
     return {"status": "ok"}
 
 
+@router.get("/api/health")
+async def api_health(response: Response) -> dict:
```

Ruff (0.14.14)

[warning] Unused function argument: `response`

(ARG001)